### PR TITLE
admin dashboard - new hawc_admin application

### PR DESCRIPTION
**Pull request by rabstejnek**
Rewrite the admin dashboard and associated views and API endpoints into a new django application. In addition to migrating code, we added a few new dashboard views. The current views include:

1. View assessment size for all assessments for key object counts
2. View top 20 assessments with growth in the last 6 months
3. Item creation timeline for a select assessment
4. Object growth over time
5. User insights dashboard

### CODE REVIEW

File: frontend/shared/utils/tryWebAppStartup.js
Comment (rabstejnek) [inline frontend/shared/utils/tryWebAppStartup.js:10]: This change was necessary for `startup` to work correctly on admin pages, since an element with id `main` already existed on those templates. However, subsequent commits to this PR effectively removes `startup` from admin pages, but I think this change should remain in just to make it more flexible. Let me know if you disagree and I'll revert it!

File: hawc/templates/base.html
Comment (shapiromatron) [inline hawc/templates/base.html:108]: moved this method from base.html to the `frontend/shared/startup.js` method; consistent with jQuery ajax

File: hawc/apps/myuser/views.py
Comment (shapiromatron) [inline hawc/apps/myuser/views.py:61]: additional change; prevent trying to relogin if already authenticated

File: hawc/apps/hawc_admin/urls.py
Comment (shapiromatron) [inline hawc/apps/hawc_admin/urls.py:15]: not my favorite pattern (or best work), but this makes it relatively simple to pass a list of patterns to add to django routes which is determined by the admin setting.

**Review by shapiromatron — APPROVED** [review]
@rabstejnek sorry, I merged in a side-branch I had into your work b/c I thought it was related. However when I got into it, it seemed like we kept bolting on more admin-type things to the assessment package, and we needed to split it out into its own app. I took a pass at this.

Let me know if everything works ok for you before merging, or if have any comments for me...

File: frontend/shared/utils/tryWebAppStartup.js
Comment (shapiromatron) [reply frontend/shared/utils/tryWebAppStartup.js:10]: this seems reasonable to me; I bet it'll come in handy...

**Comment by rabstejnek** [comment]
@shapiromatron Everything works, it looks good! I'm a fan of refactoring all of the admin stuff out of assessment and into its own app too. I approve if its ready for merge.

**Pull request merged**
